backend/app/analysis/foreground_percentage.py

```diff
@@ -36,8 +36,17 @@ def analyze_numpy_photo(photo):
     Returns percentage of a numpy photo that is part of the foreground
     """
     foreground_mask = grabcut_analysis(photo)
+
+    black_pixels = []
+    for i in range(0, len(foreground_mask), 20):
```

Comment:
Not clear to me why we're limiting this range -- can you please comment and explain?

Reply:
plus label any magic numbers...